ntbs-service/Pages/Notifications/Edit/Patient.cshtml.cs

```diff
@@ -71,6 +85,11 @@ protected override async Task ValidateAndSave()
                 ModelState.ClearValidationState("Patient.YearOfUkEntry");
             }
 
+            if (string.IsNullOrEmpty(Patient.OccupationOther))
```

Comment:
Not sure I follow this?
Should the `if` condition not be about the value in the `select` box?

Reply:
This relies on the UpdatePatientFlags (or whatever it's called) in NotificationService having been run. Could add a comment to this effect, or otherwise attempt to refactor to make that clearer (it would mean multiple hits the db though...)